src/components/marker-chart/Canvas.js

```diff
@@ -43,7 +43,7 @@ type OwnProps = {|
   +rangeEnd: Milliseconds,
   +markerTimingRows: MarkerTimingRows,
   +rowHeight: CssPixels,
-  +markers: Marker[],
+  +markers: Marker[], // This needs to be the full marker table
```

Comment:
I think it would be better to rename this to `unfilteredMarkers`, then we don't need the comment, and it's obvious what's going on in the code. In fact, I would suggest using the variable `unfilteredMarkers` wherever we are pulling in this information.